packages/dice-webapp/components/sidebar/sidebar-data.tsx

```diff
@@ -0,0 +1,64 @@
+import { LuLayoutDashboard } from "react-icons/lu";
+import {
+  BiSolidDashboard,
+  BiMessageSquareError,
+  BiSolidMessageSquareError,
+} from "react-icons/bi";
+import {
+  PiCertificate,
+  PiCertificateFill,
+  PiUserBold,
+  PiUserFill,
+  PiClockCountdown,
+  PiClockCountdownFill,
+} from "react-icons/pi";
+import {
+  AiOutlineSafetyCertificate,
+  AiFillSafetyCertificate,
+} from "react-icons/ai";
+
+export type SidebarItem = {
+  title: string;
+  path?: string;
+  defaultIcon: React.ReactNode;
+  activeIcon: React.ReactNode;
+};
+
+export const orgSidebarData: SidebarItem[] = [
```

Comment:
Add data for student sidebar and admin sidebar too

Reply:
Data items for student and admin sidebar are added.